src/Components/PaymentMethod/PaymentMethodContainer.js

```diff
@@ -1420,23 +1464,23 @@ const PaymentMethodContainerWithoutStripe = ({
           invoice?.amount_remaining ??
           getOrderItemsTotal();
 
-        if (
-          source?.card?.three_d_secure === "required" &&
-          totalAmount > 0
-        ) {
-          return resolveTaxCalculation().then((res) =>
-            generate3DSecureSource(
-              source,
-              res?.totalAmountWithTax ?? totalAmount
-            ).then(({ source, error }) => {
-              if (error) {
-                return handlePaymentError(error);
-              }
-
-              toggleAuthenticationPendingView(true, source);
-            })
-          );
-        }
+        // if (
+        //   source?.card?.three_d_secure === "required" &&
+        //   totalAmount > 0
+        // ) {
+        //   return resolveTaxCalculation().then((res) =>
+        //     generate3DSecureSource(
+        //       source,
+        //       res?.totalAmountWithTax ?? totalAmount
+        //     ).then(({ source, error }) => {
+        //       if (error) {
+        //         return handlePaymentError(error);
+        //       }
+
+        //       toggleAuthenticationPendingView(true, source);
+        //     })
+        //   );
+        // }
```

Comment:
@Shaker-Pelcro  Please don't leave commented out code in the source. Remove it entirely instead.

Reply:
OK